src/tradingtimes/tradingTimes.es6

```diff
@@ -144,11 +143,13 @@ const initTradingWin = ($html) => {
 
       /* refresh the table with result of {trading_times:yyyy_mm_dd} from WS */
       const refresh = (data) => {
-        const result = processData(menu.extractFilteredMarkets(data));
+        const result = processData(menu.extractFilteredMarkets(data[1]));
+        const header = getMarketsSubmarkets(data[0].active_symbols);
+
```

Comment:
Think we can add a `const ... = Object.keys(header)` here and reuse it below at line 161 and 167